.ci/lib.sh

```diff
@@ -30,6 +30,10 @@ function clone_and_build() {
 
 	pushd ${project_dir}
 
+	# Ensure HEAD is at latest master.
```

Comment:
emm nop, doing this you override the pull request code fetched by semaphore or any other CI

Reply:
Ah yes you're right...